Colours.txt

```diff
@@ -38,6 +38,7 @@
 #define JJOCAS 6249039
 
 ;SMR Colours
+#define smrKRRunup 9352594
```

Comment:
We keep colours in alphabetical order within the sections, so this one would go after `smrKKBlue`.

Reply:
@kristiankunc You have marked this as resolved, but haven't made the change.

Reply:
@PLM1995 Sorry, forgot to commit, resolved now.